content/post/know-your-aws-community-day-india-2020-speaker-abesh-rajasekharan.md

```diff
@@ -0,0 +1,21 @@
++++
+author = "Dhruv Aggarwal"
+date = 2020-09-23T16:54:08Z
+description = "Abesh Rajasekharan will be speaking on \"Using Ansible to manage Hybrid Architecture\" at <time> on <date> at AWS Community Day India 2020."
+feature_image = "/images/abesh.PNG"
+tags = ["AWS", "AWSCommunityDay2020", "Speaker", "Ansible"]
```

Comment:
Speaker tag to be the first

Reply:
It's not able to come first. I have tried many times.

I also tried by deleting all tags and starting with speaker tag but still, it's not taking

Reply:
You can use the raw editor or clone the repo and edit the markdown page. Looks like Forestry mangles the shortcode and the youtube video is now not embedded properly

Reply:
@SathyaBhat Markdown in forestry.io doesn't allow to edit tags and cloning of repo also doesn't work. It can only be committed from forestry.io only. 
Also, please guide me on embedding youtube videos in the article. Resources and FAQs are not good enough.

Reply:
you can definitely clone the repo and edit it, thats what I did. Invited you to the team. If you still can't edit, please create a fork and send a PR.

Reply:
Updated. Created PR #7 .